src/app/styles/js/shared.js

```diff
@@ -389,6 +348,21 @@ export const HeaderTitle = styled.h3`
   `}
 `;
 
+export const StyledSubHeader = styled.h2`
+  font: ${title1};
+  font-weight: 600;
+  color: ${black54};
+  text-align: center;
+  margin-top: ${units(2)};
+`;
```

Comment:
Fine for now but would be best to use `Typography` and/or a dedicated component under `components/layout`. Please add a FIXME comment

Reply:
Added FIXME comment